string id
string type

string ACTIVATE_LEFT = activateLeft
string ACTIVATE_RIGHT = activateRight
string DEACTIVATE_LEFT = deactivateLeft
string DEACTIVATE_RIGHT = deactivateRight
string HEAD_RESET = headReset
string LEAVE_LEFT = leaveLeft
string LEAVE_RIGHT = leaveRight
string LOOK_AT = lookAt
string LOOK_FOR = lookFor
string MOVE = move
string ROTATE = rotate
string SMELL_LEFT = smellLeft
string SMELL_RIGHT = smellRight
string SPEECH = speech
string TAKE_LEFT = takeLeft
string TAKE_RIGHT = takeRight
string TASTE_LEFT = tasteLeft
string TASTE_RIGHT = tasteRight
string TURN = turn
string CANCEL_COMMANDS = cancelCommands